src/components/NavBar/NavBar.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../style/variable.scss";
+
+.nav-link {
+  color: $primary !important;
+  font-weight: bold;
+  font-size: 1.2rem;
+  margin: auto 10px;
+}
+.bg-light {
```

Comment:
for the non bootstrap classes can we use camelCase, but I guess this one is a bootstrap class right ?

Reply:
yes this one was from bootstrap , but since it was one element i added a new class name to it 
and  use it and updated the test